docs/guide.md

```diff
@@ -54,6 +54,26 @@ If an access token is not provided, the following rate limits apply:
 - 25 container builds per day
 - 250 container pulls per hour
 
+## Reproducible builds with SHA256 container image references
+
+Referring to the SHA256 hash of a container imagine is a common best practice in pipeline development. For example, the SHA256 hash for a container might be similar to `sha256:3235...ce8f`. If you use the Wave service to augment your containers, such as to add Fusion to a container, the original SHA256 hash of the image is no longer correct.
+
+To accommodate the new layers added to a container image augmented with Wave, you can use the freeze feature of Wave. In your pipeline configuration, ensure that you specify the following settings:
+
+```
+wave.enabled = true
+wave.freeze = true
+wave.strategy = ['dockerfile']
+wave.build.repository = 'docker.io/<user>/<repository>'
+```
+
+In your pipeline, you can use the SHA256 of the Wave augmented container, available in the target repository that you specify with the `wave.build.repository` parameter. For example:
```

Comment:
@pditommaso - this doesn't describe how to get this container URI. Is the suggestion that the user should use the Wave CLI to generate it? (in which case the above snippet should be updated). Or run using `nextflow inspect` to get the returned URL?

I feel like we need an extra step here to describe this, either way.

Reply:
> how to get this container URI

Which container URI?

Reply:
I think probably by running `nextflow inspect` given the context here.

Reply:
More details: with the nextflow config in the above block, Nextflow will use Wave to generate an image. Running `nextflow inspect` on a pipeline returns a list of all image URIs, which will have gone via Wave. If frozen, those images will be pushed to the registry.

The user can take those wave-augmented URIs and put them in the pipeline directly, which is the second block of text here.